SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -5,6 +5,8 @@
 #include "EngineClasses/SpatialNetDriver.h"
 #include "Utils/SpatialActorUtils.h"
 
+DEFINE_LOG_CATEGORY(LogGridBasedLBStrategy);
+
 UGridBasedLBStrategy::UGridBasedLBStrategy()
```

Comment:
Can we have a TODO to delete the magic numbers here? I don't think initializing a grid without explicit values would ever be an expected workflow so we're better off having INVALID constants (or similar) and then passing in the LoadBalancerSettings in the Init function and extracting values from there?

Reply:
Ironically, I've just added a use case (LBStrategy not specified) for having a grid initialized by default values.